sklearn/metrics/_classification.py

```diff
@@ -2062,25 +2092,36 @@ class after being classified as negative. This is the case when the
     --------
     >>> import numpy as np
     >>> from sklearn.metrics import class_likelihood_ratios
-    >>> class_likelihood_ratios([0, 1, 0, 1, 0], [1, 1, 0, 0, 0])
+    >>> class_likelihood_ratios([0, 1, 0, 1, 0], [1, 1, 0, 0, 0],
+    ...                          replace_undefined_by=np.nan)
```

Comment:
I feel like we shouldn't need this `replace_undefined_by` here since we're not changin behavior

Reply:
I got a doctest failure because of the FutureWarning, this is why I added it.